testing/code/test_terminal_writer.py

```diff
@@ -18,6 +18,7 @@ def test_code_highlight(has_markup, expected, color_mapping):
     f = StringIO()
     tw = TerminalWriter(f)
     tw.hasmarkup = has_markup
+    tw.code_highlight = True
```

Comment:
Added this with the assumption that this test assumes code is to be highlighted.